Canvas/Canvas/CanvasAppDelegate.swift

```diff
@@ -60,11 +60,12 @@ class AppDelegate: UIResponder, UIApplicationDelegate, AppEnvironmentDelegate {
 
         if let session = Keychain.mostRecentSession {
             window?.rootViewController = LoadingViewController.create()
+            window?.makeKeyAndVisible()
             userDidLogin(keychainEntry: session)
         } else {
             window?.rootViewController = LoginNavigationController.create(loginDelegate: self, fromLaunch: true)
+            window?.makeKeyAndVisible()
         }
-        window?.makeKeyAndVisible()
```

Comment:
this was causing the CanvasCore.LocalizationManager to not be able to display the restart alert when the device switched locales with a logged in user with a different locale